src/lib/utils/meta_tables/meta_system_utilization_table.cpp

```diff
@@ -71,16 +71,21 @@ uint64_t MetaSystemUtilizationTable::_get_total_time() {
 /*
  * Returns the time in ns that ALL processes have spent on the CPU
  * since an arbitrary point in the past.
+ * This might be used to differentiate between CPU time consumed by this process 
+ * and by other processes on the same machine.

```

Comment:
Could you please check your editor's line wrapping settings? We only break after 120 characters or to add structure to the text.